src/main/java/org/opentripplanner/routing/algorithm/RoutingWorker.java

```diff
@@ -96,6 +96,12 @@ public RoutingResponse route(Router router) {
         request.setRoutingContext(router.graph);
         if (request.modes.transitModes.isEmpty()) { return Collections.emptyList(); }
 
+        if (!router.graph.transitFeedCovers(request.dateTime)) {
+            throw new RoutingValidationException(Collections.singletonList(
+                    new RoutingError(RoutingErrorCode.OUTSIDE_SERVICE_PERIOD, null)
```

Comment:
This error is because the `request.dateTime` is outside the service period, so we should return a constant(enum) for that even if it is just one possibility. Add `DATE_TIME` to `InputField` and call the constructor of `RoutingError` with that.